tuf/data/roles_test.go

```diff
@@ -194,46 +194,46 @@ func TestErrInvalidRole(t *testing.T) {
 }
 
 func TestIsDelegation(t *testing.T) {
-	assert.True(t, IsDelegation(filepath.Join(CanonicalTargetsRole, "level1")))
+	assert.True(t, IsDelegation(path.Join(CanonicalTargetsRole, "level1")))
 	assert.True(t, IsDelegation(
-		filepath.Join(CanonicalTargetsRole, "level1", "level2", "level3")))
-	assert.True(t, IsDelegation(filepath.Join(CanonicalTargetsRole, "under_score")))
-	assert.True(t, IsDelegation(filepath.Join(CanonicalTargetsRole, "hyphen-hyphen")))
+		path.Join(CanonicalTargetsRole, "level1", "level2", "level3")))
+	assert.True(t, IsDelegation(path.Join(CanonicalTargetsRole, "under_score")))
+	assert.True(t, IsDelegation(path.Join(CanonicalTargetsRole, "hyphen-hyphen")))
 	assert.False(t, IsDelegation(
-		filepath.Join(CanonicalTargetsRole, strings.Repeat("x", 255-len(CanonicalTargetsRole)))))
+		path.Join(CanonicalTargetsRole, strings.Repeat("x", 255-len(CanonicalTargetsRole)))))
 
 	assert.False(t, IsDelegation(""))
 	assert.False(t, IsDelegation(CanonicalRootRole))
-	assert.False(t, IsDelegation(filepath.Join(CanonicalRootRole, "level1")))
+	assert.False(t, IsDelegation(path.Join(CanonicalRootRole, "level1")))
 
 	assert.False(t, IsDelegation(CanonicalTargetsRole))
 	assert.False(t, IsDelegation(CanonicalTargetsRole+"/"))
-	assert.False(t, IsDelegation(filepath.Join(CanonicalTargetsRole, "level1")+"/"))
-	assert.False(t, IsDelegation(filepath.Join(CanonicalTargetsRole, "UpperCase")))
+	assert.False(t, IsDelegation(path.Join(CanonicalTargetsRole, "level1")+"/"))
+	assert.False(t, IsDelegation(path.Join(CanonicalTargetsRole, "UpperCase")))
 
 	assert.False(t, IsDelegation(
-		filepath.Join(CanonicalTargetsRole, "directory")+"/../../traversal"))
+		path.Join(CanonicalTargetsRole, "directory")+"/../../traversal"))
 
 	assert.False(t, IsDelegation(
-		filepath.Join(CanonicalTargetsRole)+"///test/middle/slashes"))
+		path.Join(CanonicalTargetsRole)+"///test/middle/slashes"))
 
 	assert.False(t, IsDelegation(
-		filepath.Join(CanonicalTargetsRole)+"/./././"))
+		path.Join(CanonicalTargetsRole)+"/./././"))
```

Comment:
thanks for catching these, the `join` is definitely unnecessary so I've removed them